source/slang/hlsl.meta.slang

```diff
@@ -5285,7 +5285,7 @@ struct $(item.name)
     uint DecrementCounter();
 
     [__readNone]
-    [__unsafeForceInlineEarly]
+    [ForceInline]
```

Comment:
@csyonghe: I feel like I'm missing something.. 
`[ForceInline]` will work fine with auto-diff because we check for SPIRVAsm blocks.
It's only `[__unsafeForceInlineEarly]` that is a problem.

Reply:
Ah, sorry, you did change the line, I didn't realize this is for RWStructuredBuffer.